### ISSUE TITLE
Dump PR config changes into a PR comment.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12018]
## Description
This should allow us to determine if a Dependabot module update or similar action has post-`drush deploy` consequences for Drupal config that need intervention from a developer.

This might necessitate some cleanup or some scripting to tweak output so that Drush doesn't show us a YAML diff with lines in different places and act like it's a great revelation.

## Acceptance Criteria
- [ ] PRs get a comment after Tugboat tasks run indicating unexpected config changes.

**Comment by swirtSJW** [comment]
Yes a proper test would avoid this kind of scenerio

  Catastropic example:

1. Module bumps to new version.
2. hook_update_n for that module updates a setting
3. config import, which follows updb, wipes out the setting.
4. Devs chase flaming goose.

This one is a hard one to test for because:

-  if there is config change prior to config import runs, config import may wipe it out
- if there is data from database that does not match and creates a change to the config file, it may create a lot of false alarms and require the PR to be rebased more often on main to get a passing test.
- If you test before config import, it is too soon, if you test after config import it is too late.

**Comment by swirtSJW** [comment]
Good useful stuff here https://dsva.slack.com/archives/CT4GZBM8F/p1676563399077549?thread_ts=1676562252.659019&cid=CT4GZBM8F

TLDR is a three step intervention in deployment testing of dependabot bumps
1. Run config export dry run right after drush updb (before config import) and diff that output to github comment.
2. Run config export dry run right after drush config import and diff that output to github comment.
3. diff the .install to the previous installed version and output that to a github comment with enough noise for and engineer to see.


1.
